### PR TITLE
UTs for `commmandoverrider.go`

### PR DESCRIPTION
**Pull request by 1Shubham7**
**What type of PR is this?**

/kind test

**What this PR does / why we need it**:

I have added unit tests for the `cloud/pkg/controllermanager/edgeapplication/overridemanager/commandoverrider.go`

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cloud/pkg/controllermanager/edgeapplication/overridemanager/commandoverrider_test.go [hunk 1]
```diff
@@ -0,0 +1,836 @@
...
+
+package overridemanager
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	appsv1alpha1 "github.com/kubeedge/kubeedge/pkg/apis/apps/v1alpha1"
```
Comment (Shelley-BaoYue) [inline cloud/pkg/controllermanager/edgeapplication/overridemanager/commandoverrider_test.go:25]:
```suggestion
	appsv1alpha1 "github.com/kubeedge/kubeedge/pkg/api/apis/apps/v1alpha1"
```

**Review by Shelley-BaoYue — COMMENTED** [review]
/lgtm

**Comment by 1Shubham7** [comment]
@WillardHu please review this as well : )

**Pull request merged**